pkg/resources/etcd/statefulset.go

```diff
@@ -101,9 +102,8 @@ func StatefulSetCreator(data etcdStatefulSetCreatorData, enableDataCorruptionChe
 
 			set.Spec.Template.Spec.InitContainers = []corev1.Container{
 				{
-					Name: "etcd-launcher-init",
-
-					Image:           data.ImageRegistry(resources.RegistryQuay) + "/kubermatic/etcd-launcher:" + resources.KUBERMATICCOMMIT,
+					Name:            "etcd-launcher-init",
+					Image:           data.EtcdLauncherImage() + ":" + resources.KUBERMATICCOMMIT,
```

Comment:
How would this work? Even if I override the image via the new option, it will still append `KUBERMATICCOMMIT` to it, right ?

Reply:
Oh you will override both image and the kubermaticcommit, right ?

Reply:
Exactly. We'll have our own KUBERMATICCOMMIT values but they're determined the same way as upstream. I.e. we'll build and push the image for our fork, and KUBERMATICCOMMIT containing the sha1 of the commit will be baked into the binary and also used for the image tag name when pushing to our own docker repo.

Reply:
Apiserver image override is handled the same way: https://github.com/kubermatic/kubermatic/blob/dcd7e764b61a754522fac99dc395ccd133f2cd3c/pkg/resources/usercluster/deployment.go#L162